staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -897,7 +937,12 @@ func BuildHandlerChainWithStorageVersionPrecondition(apiHandler http.Handler, c
 }
 
 func DefaultBuildHandlerChain(apiHandler http.Handler, c *Config) http.Handler {
-	handler := filterlatency.TrackCompleted(apiHandler)
+	handler := apiHandler
+	if c.PeerProxy != nil {
```

Comment:
I expected this in kube-apiserver or kube-aggregator, not here. Was that considered?